### DISCUSSION TITLE
Illuminazione esterna

### DISCUSSION THREAD

**Discussion by gianluca891** [indomus/forum#290] [Hardware & Firmware]
Buongiorno a tutti,

avrei intenzione di realizzare un'impianto di illuminazione esterno per la tettoia di casa.
In quanto mi devo "attaccare" ad un quadro elettrico e quindi dedicare un interruttore alle luci, esiste un magnetotermico smart? se si, potrebbe essere la soluzione giusta? o è il caso di prendere in considerazione altri prodotti (sonoff, shelly, ecc..)?
Inoltre, mi servirebbe che l'accensione fosse "manuale" e "smart" ma non ho possibilità di installare eventuali placche con interruttori.

P.s. sto cercando anche delle lampadine a "bolla" con filamento, se qualcuno conoscesse qualche modello....

Grazie.

**Comment by essegi65** [comment]
Di solito in una abitazione dovresti avere già 2 circuiti separati (linea luci, linea potenza che sono le prese), io sarei per non mettere un nuovo magnetotermico solo per delle luci, penso che la loro potenza sia bassa, potresti prendere la linea luci da una scatola di derivazione e portarla a un interruttore (se c'è spazio nella scatola 503 metti lì il comando smart (sonoff mini o shelly) e poi riparti verso il punto luce nella terrazza. Se non ci dovesse essere spazio nella scatola 503 metti il comando smart nella scatola di derivazione e poi vai alla 503 e da lì prosegui verso il punto luce. Io almeno ho fatto così per i miei punti luce in casa mia.

**Reply by gianluca891** [reply]
in verità i circuiti in partenza sono 3 (linea luci, linea potenza e una terza linea per il giardino). Fuori c'è un secondo quadro con una linea potenza, linea luci giardino e devo implementare una linea luci per la tettoia, in quanto quelle dei giardino hanno un temporizzatore. 
Purtroppo non ho scatole 503 con interruttori, solo quel quadro e non posso aggiungerla, per quello dovrò comandare tutto da un magnetotermico.

Spero di esser riuscito a dar un'idea di come è fatto l'impianto.

Quindi il problema è metto un comando smart prima del magnetotermico? lo metto dopo con il rischio che venga acceso e spento in continuazione? hanno problemi a stare in esterno i comandi smart?

**Reply by essegi65** [reply]
Visto come dici io farei una deviazione sulla linea luci esterna prima del temporizzatore dei lampioni e li metterei il comando smart. Se proprio vuoi prima del comando smart inserisci un interruttore magnetotermico, di solito questo serve per "proteggere" l'impianto a monte.
Io all'esterno (in una scatola stagna, ovviamente) ho 2 sonoff per comandare una presa e una lampada.

**Reply by Maspar79** [reply]
Ciao essegi65, non condivido il tuo pensiero "io sarei per non mettere un nuovo magnetotermico solo per delle luci, penso che la loro potenza sia bassa..." sarebbe infatti buona norma non solo mettere uno o più magnetotermici (in base al numero di accensioni) per le luci esterne ma mettere anche un differenziale dedicato a monte di tali magnetotermici, in caso di guasti, come dispersioni, peraltro molto frequenti in circuiti esterni, in questo modo si riuscirebbe facilmente a risalire a quale ciurcuito ha causato il guasto, la bassa potenza è un altro discorso.

**Reply by gianluca891** [reply]
Buongiorno, il problema è che non avendo scatole 503 con interruttori fisici per l'accensione e spegnimento manuali, sono costretto a mettere dei magnetotermici.
